osu.Framework.Font/osu.Framework.Font.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
include this one.

Reply:
I don't know what has changed here

Reply:
🤔
Which IDE are you using?

Reply:
vscode, but the C# extension didn't work with my folder layout, so it was pretty much just a text editor with no C# functionality.

Reply:
Guess it's time to try the rider
PM me if you are interesting